src/System.Drawing.Common/src/System/Drawing/Internal/SystemColorTracker.cs

```diff
@@ -65,18 +66,18 @@ private static void CleanOutBrokenLinks()
 
         // Basic idea is to find a broken reference on the left side of the list, and swap it with
         // a valid reference on the right
-        int right = list.Length - 1;
+        int right = s_list.Length - 1;
         int left = 0;
 
-        int length = list.Length;
+        int length = s_list.Length;
 
         // Loop invariant: everything to the left of "left" is a valid reference,
         // and anything to the right of "right" is broken.
         while (true)
         {
-            while (left < length && list[left].Target != null)
+            while (left < length && s_list[left].TryGetTarget(out ISystemColorTracker? target) && target is not null)
```

Comment:
```suggestion
            while (left < length && s_list[left].TryGetTarget(out _))
```

The additional null check isn't necessary